modules/edit-modes/src/types.js

```diff
@@ -85,15 +85,22 @@ export type Tooltip = {
   text: string
 };
 
-export type EditHandleType = 'existing' | 'intermediate' | 'snap-source' | 'snap-target';
+export type EditHandleType =
+  | 'existing'
+  | 'intermediate'
+  | 'snap-source'
+  | 'snap-target'
+  | 'scale'
+  | 'rotate';
 
 export type EditHandleFeature = FeatureWithProps<
   Point,
   {
     guideType: 'editHandle',
     editHandleType: EditHandleType,
     featureIndex: number,
-    positionIndexes: number[]
+    positionIndexes: number[],
+    index?: number
```

Comment:
Looks like this is just used for scale mode. What do you think of just reusing `positionIndexes` instead of adding another property?

Also, noticed that `positionIndexes` should be nullable.

```
guideType: 'editHandle',
  editHandleType: EditHandleType,
  featureIndex: number,
  positionIndexes?: number[]
```